cpp/src/arrow/compute/kernels/vector_sort.cc

```diff
@@ -156,7 +161,7 @@ struct ChunkedArrayResolver {
   }
 
   int64_t num_chunks_;
-  const Array* const* chunks_;
+  const std::vector<const Array*> chunks_;
```

Comment:
Was there an issue with the original (chunks_ getting invalidated, presumably?)?

Reply:
Yes there was an issue, a holder for this pointer  was needed